src/ledger-bridge.ts

```diff
@@ -26,14 +26,28 @@ export type LedgerSignTypedDataResponse = Awaited<
   ReturnType<LedgerHwAppEth['signEIP712HashedMessage']>
 >;
 
+export type LedgerBridgeOptions = Record<string, unknown>;
+
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-export interface LedgerBridge {
+export interface LedgerBridge<T extends LedgerBridgeOptions> {
```

Comment:
We could remove the `eslint-disable` comment and declare a `type` instead of an `interface`, for consistency:

```suggestion
export type LedgerBridge<T extends LedgerBridgeOptions> = {
```

Reply:
resolved